internal/archiver/archiver.go

```diff
@@ -191,19 +191,28 @@ func (arch *Archiver) nodeFromFileInfo(filename string, fi os.FileInfo) (*restic
 }
 
 // loadSubtree tries to load the subtree referenced by node. In case of an error, nil is returned.
```

Comment:
The last sentence of the comment can be removed. The return value should be obvious now.

Reply:
Actually, on second thought, it's not obvious that (nil, nil) is a valid return pair. Maybe put that in?

Reply:
I've expanded the comment.